### DISCUSSION TITLE
Dependabot should warn on PRs that introduce a number of insecure dependencies

### DISCUSSION THREAD

**Discussion by devd** [community/community#29475] [Code Security] [answered]
Currently, if a new PR adds a number of insecure/out-of-date dependencies, Dependabot doesn't warn on this. But, after the PR lands, Dependabot will start warning how depedencies are out of date and even try to create a PR. This isn't a great UX: ideally, Dependabot would comment on the pull request!

**Comment by courtneycl** [comment, marked as answer]
Hi! Thanks for being here. The [dependency review action](https://github.com/marketplace/actions/dependency-review) will check your PRs and will fail if they introduce a dependency that has a known vulnerability.

**Reply by devd** [reply]
Hi! thank you! Didn't know about this. Unfortunate that it requires advanced security for private repos but I understand. Thanks!
